docs/sources/installation/mac.md

```diff
@@ -8,74 +8,43 @@ parent = "installation"
 weight = 500
 +++
 
+# Install on macOS
 
-# Installing on macOS
+This page provides instructions to help you install Grafana on macOS. 
 
-This page provides instructions to help you install Grafana on macOS. You can either install using homebrew or a binary tar file.
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
+## 1. Download and install
 
-## Install using homebrew
+Before you begin, you must have [Homebrew](http://brew.sh/) installed.
 
-Installation can be done using [homebrew](http://brew.sh/)
+1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install. 
+   * The most recent Grafana version is selected by default.
+   * The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
+2. Select an **Edition**.
+   * **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
+   * **Open Source** - Functionally identical to the enterprise version, but you will need to download the enterprise version if you want enterprise features.
+3. Click **Mac**.
+4. Copy and paste the code from the installation page into your command line and run. It follows the pattern shown below.
 
-Install latest stable:
+   ```bash
+   brew update
+   brew install grafana
+   ```
 
-```bash
-brew update
-brew install grafana
-```
-
-To start grafana look at the command printed after the homebrew install completes.
-
-To upgrade use the reinstall command
-
-```bash
-brew update
-brew reinstall grafana
-```
-
--------------
-
-You can also install the latest unstable grafana from git:
-
-
-```bash
-brew install --HEAD grafana/grafana/grafana
-```
+## 2. Start Grafana
 
-To upgrade grafana if you've installed from HEAD:
+Start Grafana using Homebrew services: 
 
 ```bash
-brew reinstall --HEAD grafana/grafana/grafana
+brew services start grafana
 ```
 
-### Start Grafana
-
-To start Grafana using homebrew services first make sure homebrew/services is installed.
-
-```bash
-brew tap homebrew/services
-```
+## Upgrade with Homebrew
 
-Then start Grafana using:
+To upgrade Grafana, use the reinstall command:
 
 ```bash
-brew services start grafana
+brew update
+brew reinstall grafana
 ```
-
-Default login and password `admin`/ `admin`
```

Comment:
Might be nitpicky, but did you intend to have the whitespace after but not before?

Reply:
Not sure what you are seeing.

Reply:
Current mac.md ends at line 51.